client/rpcserver/handlers.go

```diff
@@ -192,6 +193,22 @@ func handleNewWallet(s *RPCServer, params *RawParams) *msgjson.ResponsePayload {
 		resErr := msgjson.NewError(msgjson.RPCWalletExistsError, errMsg)
 		return createResponse(newWalletRoute, nil, resErr)
 	}
+
+	walletDef, err := asset.WalletDef(form.assetID, form.walletType)
+	if err != nil {
+		errMsg := fmt.Sprintf("error creating %s wallet: unable to get wallet definition: %v",
+			dex.BipIDSymbol(form.assetID), err)
+		resErr := msgjson.NewError(msgjson.RPCWalletDefinitionError, errMsg)
+		return createResponse(newWalletRoute, nil, resErr)
+	}
+
+	// Apply default config options if they exist.
+	for _, opt := range walletDef.ConfigOpts {
+		if _, has := form.config[opt.Key]; !has && opt.DefaultValue != nil {
+			form.config[opt.Key] = fmt.Sprintf("%v", opt.DefaultValue)
```

Comment:
`fmt.Sprintf("%v", ...)` should do the trick for all the types we give in `DefaultValue`, but in our other uses of this field (`WalletConfigForm.map`) we consider the other flags like `isdate`and `repeatable`.  I don't feel like we need to replicate all that at this time though.

Reply:
To clarify, nothing to change unless another reviewer sees an issue related to this.